internal/server/audit/checker_test.go

```diff
@@ -0,0 +1,56 @@
+package audit
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestChecker(t *testing.T) {
+	testCases := []struct {
+		name          string
+		eventPairs    []string
+		expectedError error
+		pairs         map[string]bool
+	}{
+		{
+			name:          "wild card for nouns",
+			eventPairs:    []string{"*:create"},
+			expectedError: nil,
+			pairs: map[string]bool{
+				"flag:create":       true,
+				"constraint:create": true,
+				"namespace:update":  false,
+			},
```

Comment:
pedant comment, but could we make this exhaustive? Just drop all the expected matches in here.
Since there is only a handful.